CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 
 As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
 
-We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, age, or religion.
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion.
```

Comment:
@TheReverend403 Well, it's actually happened a few times here since this drama started, but I don't think I've ever seen any instances before that. GitHub issues tend to be focused on code, not people.

For this reason, I think I actually agree that a CoC is unnecessary. On the other hand though, I don't really see it doing any harm either as long as we word it correctly, so I'm not really opposed to the idea either.